system-setup.py

```diff
@@ -108,6 +107,7 @@ def common_last(self):
         self.pip_install("--no-cache-dir git+https://github.com/Grokzen/redis-py-cluster.git@master")
         self.pip_install("--no-cache-dir git+https://github.com/RedisLabsModules/RLTest.git@master")
         self.pip_install("--no-cache-dir git+https://github.com/RedisLabs/RAMP@master")
+        self.pip_install("jinja2")
```

Comment:
What is this lib?

Reply:
It's Python templating lib, needed to process ramp.yml via the readies xtx utility.